Slim/Player/ReplayGain.pm

```diff
@@ -198,7 +198,7 @@ sub trackSampleRateMatch {
 	my $offset = shift;
 
 	my ($current_track, $compare_track) = $class->findTracksByIndex($client, $offset);
-	return if (!$current_track || !$compare_track);
+	return 1 if (!$current_track || !$compare_track);
 
 	if (!blessed($current_track) || !blessed($compare_track)) {
```

Comment:
Should we change the `return 0` inside this condition, too? It's basically the same condition as above.

Reply:
I would think so but I'm wondering in what case a track could not be a class object. Is there a case where it could be a simple reference for a plugin or does it always have to belong to a class? I don't know LMS well enough to answer that. If it is the case, then I assume that it's just a safety net to disable crossfade "in case". 
BTW, in general, I would have disabled the fade mode *only* if it is crossfade. To me, all other modes (in, out, in_out) are valid even if sample rates differ. The difficulty is to cross-mix track with different sample rate